### ISSUE TITLE
C benchmark PingPong is broken

### ISSUE THREAD

**Issue by cmnrd** [lf-lang/lingua-franca#1058]
The C version of PingPong prints the following error message and returns with a non-zero exit code.
```
Pong expected to receive 1000000 inputs, but it received 12000000.
```

The benchmark implementation looks correct to me, and it used to work in the past. Therefore, I assume that there is a bug in the code generator and/or the c runtime.

**Comment by petervdonovan** [comment]
This is fixed in the [`benchmark-updates` branch](https://github.com/lf-lang/benchmarks-lingua-franca/tree/benchmark-updates). It is a bug in the benchmark. I just didn't merge it because I figured that more problems would probably turn up.

**Comment by cmnrd** [comment]
I see. Should we get this branch merged then? Do you want to open a PR?

**Comment by cmnrd** [comment]
Closed by lf-lang/benchmarks-lingua-franca#3
